Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -6972,16 +6972,21 @@ bool CStaticFunctionDefinitions::GetKeyState(const char* szKey, bool& bState)
 
     CKeyBindsInterface* pKeyBinds = g_pCore->GetKeyBinds();
     const SBindableKey* pKey = pKeyBinds->GetBindableFromKey(szKey);
+    unsigned long keyCode = 0;
+
     if (pKey)
+        keyCode = pKey->ulCode;
+    else if (strcmp(szKey, "capslock") == 0)
+        keyCode = VK_CAPITAL;

```

Comment:
since `VK_CAPITAL` is a bindable key, will this ever execute?

Reply:
yeah, it does

Reply:
Why? `capslock` is listed here https://wiki.multitheftauto.com/wiki/Key_names and you've stated it's a bindable key, so this should not executed.

Reply:
Yeah, just seen, sec